src/koop/marklogic.js

```diff
@@ -58,6 +58,9 @@ MarkLogic.prototype.getData = function getData (req, callback) {
 
 	  new MarkLogicQuery().providerGetData(providerRequest, dbClient)
 	    .then(data => {
+        if (req.query.skipFiltering === true) {
```

Comment:
Posted a question internally, but just for reference - TIL that this is where the settings for "data" in https://github.com/koopjs/koop/tree/master/packages/featureserver can be applied.

Reply:
Based on what @jkerr5 is saying, we can't do this because it results in projection=true, and GDS never does that (I don't quite understand that yet, only that it means we can't do "all" for filtering).